git/rev_list_scanner_test.go

```diff
@@ -160,7 +160,7 @@ func TestRevListArgs(t *testing.T) {
 
 func TestRevListScannerCallsClose(t *testing.T) {
 	var called uint32
-	err := errors.New("Hello world")
+	err := errors.New("hello world")
```

Comment:
I don't have a strong opinion on this one, but it is customary to write "Hello world" with an initial capital.

Reply:
Completely changed error text here